.circleci/config.yml

```diff
@@ -178,6 +178,24 @@ jobs:
           command: yarn test:coverage
           working_directory: packages/contracts-periphery
 
+  contracts-governance-tests:
+    docker:
+      - image: ethereumoptimism/js-builder:latest
+    resource_class: xlarge
+    steps:
+      - restore_cache:
+          keys:
+            - v2-cache-yarn-build-{{ .Revision }}
+      - checkout
+      - run:
```

Comment:
Should we also add slither?

Reply:
slither isn't configured for this package and I don't feel like setting it up, not much dev will happen here and we could consider consolidating this package into others in the future